packages/image_picker/ios/Classes/ImagePickerPlugin.m

```diff
@@ -150,6 +166,11 @@ - (void)imagePickerController:(UIImagePickerController *)picker
                                   details:nil]);
     }
   } else {
+    NSURL *assetURL = [info objectForKey:UIImagePickerControllerReferenceURL];
+    PHFetchResult<PHAsset *> *result = [PHAsset fetchAssetsWithALAssetURLs:@[ assetURL ]
```

Comment:
`assetURL` can be nil if user take a fresh picture in the image_picker, which leads to crash here.